### ISSUE TITLE
UUID version issue

### ISSUE THREAD

**Issue by jakethesquid** [stabacco/stake-python#123]
Hi,

Big fan of this API :)

I'm currently having an issue where any attempt to access the account causes the following error. The code was previously running fine, but stopped working about 2 days ago (without any changes made). 
Any idea what would cause this?    

my_equities = await stake_session.equities.list()
  File "C:\Users\xxx\.conda\envs\stocks\lib\site-packages\stake\equity.py", line 66, in list
    return EquityPositions(**data)
  File "pydantic\main.py", line 400, in pydantic.main.BaseModel.__init__
pydantic.error_wrappers.ValidationError: 1 validation error for EquityPositions
equityPositions -> 8 -> instrumentID
  uuid version 4 expected (type=value_error.uuid.version; required_version=4)

Cheers,
Jake

**Comment by stabacco** [comment]
It's possible that the equity response has changed its format on the server
side. It happens sometimes. I'll have a look shortly

**Comment by stabacco** [comment]
I'm having trouble reproducing it, I've run this code (from the readme) and it seems to work fine on my portfolio.

```import stake
import asyncio


async def show_portfolio():
    # here the client will use the STAKE_TOKEN env var for authenticating
    async with stake.StakeClient() as stake_session:
        my_equities = await stake_session.equities.list()
        for my_equity in my_equities.equity_positions:
            print(my_equity.symbol, my_equity.yearly_return_value)
        return my_equities

asyncio.run(show_portfolio())
```
Seems like the equity number 8 in your list has an  instrumentID  which is not a UUID. Can you figure out which symbols it is?

**Comment by jakethesquid** [comment]
That code has the same issue.
It's now saying equity 5 is the issue (portfolio change over night). If it is ordered the same as per the Stake dashboard, it would be PM.
Let me know if that helps or if you need more info.

**Comment by stabacco** [comment]
Try this command on your shell (or equivalent in postman) 
```bash
curl 'https://global-prd-api.hellostake.com/api/users/accounts/v2/equityPositions' \
  -H 'authority: global-prd-api.hellostake.com' \
  -H 'accept: application/json' \
  -H 'stake-session-token: <YOURTOKEN>' \
  --compressed > equities.json
```

and inspect the `instrumentID` values. Does any of it look fishy?

**Comment by jakethesquid** [comment]
Nothing really jumps out, all of the UUIDs are the same format. (xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx)
Only things worth noting; when the file is opened in VIM, the text colours appear to stop half way through the PM line (at the name field).  Can't see why. This doesn't seem to happen on notepad++.

**Comment by jakethesquid** [comment]
Ok I think I found something, the version bit (bit 13) on all of them is 4, except the last one (symbol: XOM), which it is 1. All of the variant bits appear to be different.

**Comment by stabacco** [comment]
You can test uuids here https://www.freecodeformat.com/validate-uuid-guid.php . 
The XOM one does not seem to be a valid UUID4 unfortunately, hence the error. I guess we have to allow for those IDs as well.

**Comment by jakethesquid** [comment]
The website says its Valid, version 1, variant: DCE 1.1, ISO/IEC 11578:1996.

The others are all version 4. 

Looks like thats the culprit.

**Comment by stabacco** [comment]
Should be fixed in 0.2.9
